### PR TITLE
reword restart page

### LINKED ISSUE #1510: Page to restart your browser after installing Test Pilot feels like a dead end.
After restarting the browser the user is shown a page with a message asking them to restart the browser in order to choose their features.

screen shot 2016-10-03 at 3 11 24 pm

Once the browser restarts the icon is added to the toolbar, but there is no indication that anything is new.

screen shot 2016-10-03 at 3 18 04 pm

Some ideas to improve the UX here (not all my be possible):
- Have some imagery/text on the Restart Your Browser page to educate the use to the fact that the icon will be added after they restart.
- Maybe a button on the page to initiate the restart would be helpful
- Pop a Test Pilot tab after the browser has restarted and the Add-on has been installed. Can this be the active tab?

### PR DESCRIPTION
**Pull request by johngruen**
Fixes #1510

### CODE REVIEW

**Comment by johngruen** [comment]
Fixes #1510

File: frontend/src/app/components/Restart.js [hunk 1]
```diff
@@ -13,30 +13,21 @@ export default class Restart extends React.Component {
...
     return (
       <div className="full-page-wrapper space-between">
-        <Header {...this.props} forceHideSettings={ false } />
-        <div className="centered-banner restart-message">
-          <p data-l10n-id="restartRequiredSubHeader">Almost done . . .</p>
-          { this.renderSubtitle() }
+        <Header {...this.props} forceHideSettings={ true } />
+        <div className="split-banner restart-message responsive-content-wrapper">
+          <div className="restart-image">
+          </div>
```
Comment (chuckharmston) [inline frontend/src/app/components/Restart.js]: Use an `` or `<picture>` tag here? This is a content image, we shouldn't be hiding it as a background image.

File: frontend/src/app/components/Restart.js [hunk 2]
```diff
@@ -13,30 +13,21 @@ export default class Restart extends React.Component {
     });
   }
 
-  renderSubtitle() {
-    const { experimentTitle } = this.props;
-    if (experimentTitle) {
-      return (
-        <h2 data-l10n-id="restartRequiredFromExperiment">
-           data-l10n-args={JSON.stringify({ experimentTitle })}
```
Comment (chuckharmston) [inline frontend/src/app/components/Restart.js]: We do [some work](https://github.com/mozilla/testpilot/search?utf8=%E2%9C%93&q=experimentTitle) to make `experimentTitle` available here. If we won't use it anymore, let's pull all that out.

File: locales/en-US/app.ftl [hunk 3]
```diff
@@ -122,6 +122,11 @@ retirePageHeadline = Thanks for flying!
 retirePageMessage = Hope you had fun experimenting with us. <br> Come back any time.
 retirePageSurveyButton = Take a quick survey
 
+restartIntroLead = Preflight checklist
+restartIntroOne = 1. Restart your browser
```
Comment (chuckharmston) [inline locales/en-US/app.ftl]: We should be putting this in an ordered list rather than putting the bullets here.

File: locales/en-US/app.ftl [hunk 4]
```diff
@@ -122,6 +122,11 @@ retirePageHeadline = Thanks for flying!
 retirePageMessage = Hope you had fun experimenting with us. <br> Come back any time.
 retirePageSurveyButton = Take a quick survey
 
+restartIntroLead = Preflight checklist
```
Comment (chuckharmston) [inline locales/en-US/app.ftl:130]: These are different than the strings in the component. They should probably be the same.

**Review by chuckharmston — CHANGES_REQUESTED** [review]
Tests are also failing and will need updated.

**Comment by johngruen** [comment]
@chuckharmston check it out?

File: frontend/src/app/reducers/addon.js [hunk 5]
```diff
@@ -22,12 +22,11 @@ const disableExperiment = (state, { payload: experiment }) => {
   return { ...state, installed: newInstalled };
 };
 
-const requireRestart = (state, { payload: experimentTitle }) => {
+const requireRestart = (state) => {
```
Comment (chuckharmston) [inline frontend/src/app/reducers/addon.js]: No need for the parens around `state` if there's only one argument.

File: frontend/src/app/components/Restart.js [hunk 6]
```diff
@@ -13,30 +13,22 @@ export default class Restart extends React.Component {
...
     return (
       <div className="full-page-wrapper space-between">
-        <Header {...this.props} forceHideSettings={ false } />
-        <div className="centered-banner restart-message">
-          <p data-l10n-id="restartRequiredSubHeader">Almost done . . .</p>
-          { this.renderSubtitle() }
+        <Header {...this.props} forceHideSettings={ true } />
+        <div className="split-banner restart-message responsive-content-wrapper">
+          <div className="restart-image">
+            <img src="/static/images/restart-graphic@2x.jpg" width="208px" height="273px"/>
```
Comment (chuckharmston) [inline frontend/src/app/components/Restart.js]: `width` and `height` attributes are unitless.

**Review by chuckharmston — APPROVED** [review]
Few more tweaks, but it looks good. I'll approve so you can self-merge once you've addressed them.

**Comment by johngruen** [comment]
Hey @chuckharmston, i did a rebase, so you might want to give this one quick check & merge.

**Comment by johngruen** [comment]
errr, need to rebase again

**Comment by lmorchard** [comment]
Rebase looks good, previous approval, so I'll just merge

**Pull request merged**